src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -0,0 +1,165 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Optional;
+import java.util.Properties;
+
+/**
+ * A client that can create accounts, get accounts, get balances, and send ILP payments on a connector.
+ */
+public class DefaultIlpClient implements IlpClientDecorator {
+
+    private static String XPRING_ILP_GRPC_URL;
+    private static final Logger logger = LoggerFactory.getLogger(DefaultIlpClient.class);
+
+    // Bring in configuration properties such as GRPC url
+    static {
+        String propertiesFileName = "client.properties";
+        Properties properties = new Properties();
+
+        try {
+            // Make sure that the configuration file exists
+            URL res = DefaultIlpClient.class.getClassLoader().getResource(propertiesFileName);
+            InputStream inputStream = new FileInputStream(res.getFile());
+            // load the properties file
+            properties.load(inputStream);
+
+            XPRING_ILP_GRPC_URL = properties.getProperty("ilp.grpc.url");
+        } catch (Exception e) {
+            logger.info(String.format("Problem loading config file named %s. Using default configuration.", propertiesFileName));
+            XPRING_ILP_GRPC_URL = "http://localhost:6565";
+        }
+    };
+
+    private final BalanceServiceGrpc.BalanceServiceBlockingStub balanceServiceStub;
+    private final AccountServiceGrpc.AccountServiceBlockingStub accountServiceStub;
+    private final IlpOverHttpServiceGrpc.IlpOverHttpServiceBlockingStub ilpOverHttpServiceStub;
+
+    /**
+     * No-args Constructor.
+     */
+    public DefaultIlpClient() {
+        this(ManagedChannelBuilder
+          .forTarget(XPRING_ILP_GRPC_URL)
+          // Let's use plaintext communication because we don't have certs
+          // TODO: Use TLS!
+          .usePlaintext()
+          .build()
+        );
+    }
+
+    @VisibleForTesting
+    public DefaultIlpClient(String grpcUrl) {
+        this(ManagedChannelBuilder
+          .forTarget(grpcUrl)
+          // Let's use plaintext communication because we don't have certs
+          // TODO: Use TLS!
+          .usePlaintext()
+          .build()
+        );
+    }
+
+    public DefaultIlpClient(final ManagedChannel channel) {
+        this.balanceServiceStub = BalanceServiceGrpc.newBlockingStub(channel);
+        this.accountServiceStub = AccountServiceGrpc.newBlockingStub(channel);
+        this.ilpOverHttpServiceStub = IlpOverHttpServiceGrpc.newBlockingStub(channel);
+    }
+
+    @Override
+    public CreateAccountResponse createAccount() throws XpringException {
+        return this.createAccount(CreateAccountRequest.newBuilder().build(), Optional.empty());
+    }
+
+
+    @Override
+    public CreateAccountResponse createAccount(io.xpring.ilp.CreateAccountRequest createAccountRequest, Optional<String> bearerToken) throws XpringException {
```

Comment:
I want to be sure that we make it clear to outside developers that this method will _probably_ go away (creating accounts is likely going to be something developers do while signed-in to the dev portal, though this is still being debated). 

How about we mark this as `@Experimental` (see Guava) or something so that outsiders don't start relying on this as part of the actual API for XpringSDK+ILP (and also so that we all know this is something we should really think hard about before going live with).

Reply:
Does Guava have `@Experimental`? Can't find it anywhere

Reply:
Good call - I think it's actually [@Beta](https://github.com/lowasser/guava-experimental/blob/master/guava/src/com/google/common/annotations/Beta.java)

Reply:
Going to mark `getAccount` as `@Beta` as well.